README.md

```diff
@@ -0,0 +1,564 @@
+[![jazzband](https://jazzband.co/static/img/badge.svg)](https://jazzband.co/)
```

Comment:
Plz make all the link and image URLs detached so that it's easier to follow the syntax when these things are smashed together. Decoupling them also makes it possible to reuse some of the references and maintain them in a detached manner.